### ISSUE TITLE
NTR: mitochondrion outer membrane-hibernating ribosome tether activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#29131]
Please provide as much information as you can: 

* **Suggested term label:**

NTR: mitochondrion-hibernating ribosome tether activity

* **Definition (free text)**

The binding activity of a molecule that brings together a hibernating cytosolic ribosome  with the mitochondrial outer membrane, to establish or maintain the localization of the hibernating ribosome.

* **Reference, in format PMID:#######**

PMID:39379376
Title | Ribosomes hibernate on mitochondria during cellular stress.
https://europepmc.org/article/MED/39379376#free-full-text
pretty electron tomography

* **Gene product name and ID to be annotated to this term**

https://www.pombase.org/gene/SPAC6B12.15

* **Parent term(s)**

GO:0043495 | protein-membrane adaptor activity

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

* **Any other information**

We then used in situ cryo-ET to assess whether ribo- somal tethering to mitochondria was affected in the Δcpc2 strain. The tomograms showed fragmented circular mitochondria, as observed in WT cells, while no ribosome tethering was observed after 7 days of cells growing at low glucose concentrations (Fig. 5c, d, Supplementary Fig. 10 and Supplementary Movie 3).
Therefore, our data reveals a distinct mode of interaction between ribosomes and mitochondria, which is disrupted by the deletion of Cpc2. Our results indicate that Cpc2 is implicated in both cell viability and ribosome tethering to mitochondria under glucose depletion conditions, emphasizing its key role in mitochondrial homeostasis.

**Comment by ValWood** [comment]
I guess for this I can use
GO:0043495 | protein-membrane adaptor activity
but I would have no way to specify the mitochondrial outer membrane? and it isn't an individual protein

**Comment by hattrill** [comment]
If we made the term:
 'protein-outer mitochondrial membrane adaptor activity' this could be applied to a number of proteins (There is one that tethers active ribosomes that I've come across, but can't think of it just now).

**Comment by pgaudet** [comment]
Good point, we dont have an activity that describes a single gene product.
